src/itest/java/com/xebialabs/xlrelease/ci/XLReleaseNotifierFormITest.java

```diff
@@ -100,10 +98,9 @@ public void shouldShowListOfTemplatesWithSavedAsPreSelected() throws Exception {
         project.getPublishersList().add(before);
 
         HtmlForm xlrForm = jenkins.createWebClient().getPage(project, "configure").getFormByName("config");
-        HtmlSelect templateSelect = xlrForm.getSelectByName("_.template");
+        HtmlInput templateSelect = xlrForm.getInputsByName("_.template").get(0);
 
-        assertThat(templateSelect.getOptionSize(), greaterThan(1));
-        assertThat(templateSelect.getSelectedOptions().get(0).asText(), equalTo(TEMPLATE_NAME));
+        assertThat(templateSelect.asText(), equalTo(TEMPLATE_NAME));
```

Comment:
The test is not checking modified functionality, can we check if correct value is selected from auto complete